spec/tunes/tunes_stubbing.rb

```diff
@@ -177,6 +177,16 @@ def itc_stub_valid_update
          to_return(status: 200, body: itc_read_fixture_file('update_app_version_success.json'), headers: {'Content-Type' => 'application/json'})
 end
 
+def itc_stub_app_version_ref
+  stub_request(:get, "https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/ra/apps/version/ref").
+      to_return(status: 200, body: itc_read_fixture_file('app_version_ref.json'), headers: {'Content-Type' => 'application/json'})
```

Comment:
Line is too long. [130/80]<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.